tests/ui/async-await/pin-ergonomics/borrow-mut-xor-share.rs

```diff
@@ -0,0 +1,59 @@
+#![feature(pin_ergonomics)]
+#![allow(dead_code, incomplete_features)]
```

Comment:
This is preexisting, but it probably doesn't make a lot of sense that we put `pin-ergonomics` under the `async-await` directory.  In some other PR, we might want to think about moving that.

Reply:
Just move the whole `pin-ergonomics` directory elsewhere?

Reply:
Right.  Probably just moving it up one level would work, in a separate PR.

Reply:
Right.  Probably just moving it up one level would work.  Maybe it would actually be OK to do it in this PR as long as it's in a separate commit.